### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by Heydon**
* Checklist stuff (#61)
* Updated to use new Promo markup

### CODE REVIEW

**Comment by micmath** [comment]
Thank you!

**Comment by Heydon** [comment]
@micmath Sorry, bit unclear on my part, but _this_ has a lot of the generic updates in terms of terminology, fixed links etc: https://github.com/bbc/gef-docs/pull/68

File: docs/components/carousels/index.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
             <div class="gel-layout__item gel-2/5@l" style="padding-right: 12px;">
                 <h1 class="gefdocs-header__title">Carousels</h1>
-                <p class="gefdocs-header__summary gel-great-primer">A carousel is a great way to view lots of content in a limited amount of vertical space. It presents a window to view some of a row of content, while indicating there is more out of view that can be scrolled to.</p>
+                <p class="gefdocs-header__summary gel-great-primer">A Carousel is a way to browse lots of content in a limited amount of vertical space, by scrolling a window onto that content</p>
```
Comment (EmmaJP) [inline docs/components/carousels/index.html]: Is it always vertical space that is limited?

Comment (Heydon) [reply docs/components/carousels/index.html]: @EmmaJP. Hi, Emma. This has been merged, so maybe open an issue.

**Pull request merged**